### ISSUE TITLE
The effect of Intra-transfer seems to be minimal or even negative.

### ISSUE THREAD

**Issue by Sungden** [edwin465/SSVEP-stCCA#3]
Dear author, I found that the effect of intra-transfer seems minimal or even negative. In line 235 in fun_stcca.m, we can set to use only intra-transfer, inter-transfer, or both, and I got the accuracy results below. Is it right?
<img width="386" alt="image" src="https://github.com/edwin465/SSVEP-stCCA/assets/54644867/5a9923c3-7ee9-4159-86f7-dc6a4ff2d55b">

<img width="393" alt="image" src="https://github.com/edwin465/SSVEP-stCCA/assets/54644867/5eae25d5-9569-4855-a5cc-d71afa222d8d">

**Comment by edwin465** [comment]
Thanks for your question. While I check the code in line 235, **I guess that you are testing the accuracy when the feature iTR is calculated by using either r1, r2, or both them**. 

This is an interesting point. We can see which part is more important for the detection. Actually, we did not test such cases before. I do not know whether your results are correct or not. However, if your results are correct, it seems that we might find out something interesting.

By the way, I did not know why you call them as 'only inter' and 'only intra' because both of features r1 and r2 are calculated using the intra-subject transfered spatial filter. Maybe the major difference is whether they utilize the inter-subject transfered template or not.
